tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -317,9 +322,12 @@ tensorflow::Status ConvertCalibGraphToInferGraph(
       tensorflow::GraphConstructorOptions(), graph_def, &graph));
   //  get calib nodes
   std::vector<tensorflow::Node*> calib_nodes;
-  for (auto node : graph.op_nodes()) {
+  std::vector<tensorflow::Node*> topo_order;
+  tensorflow::GetPostOrder(graph, &topo_order);
```

Comment:
Why does the order matter here?

Reply:
When we are inserting TRTEngine Ops, we are rewiring its input and outputs. If the destination of an output edge of the engine is absorbed in another engine, rewiring would not work since the output node is removed. By going in reverse topological order, we ensure that the edge is updated to point to appropriate port of the downstream TRTEngine.